hare.spec

```diff
@@ -77,6 +78,8 @@ rm -rf %{buildroot}
 %{_sharedstatedir}/hare/
 %{_localstatedir}/motr/hax/
 /opt/seagate/cortx/hare/*
+/etc/logrotate.d/hare
```

Comment:
Using `%{_sysconfdir}` instead of explicit `/etc` is more idiomatic spec file syntax.

Reply:
okay, let me try.

Reply:
Done.